tests/SqlFormatterTest.php

```diff
@@ -25,65 +24,63 @@
 
 final class SqlFormatterTest extends TestCase
 {
-    private SqlFormatter $formatter;
-
-    protected function setUp(): void
-    {
-        // Force SqlFormatter to run in non-CLI mode for tests
-        $highlighter = new HtmlHighlighter();
-
-        $this->formatter = new SqlFormatter($highlighter);
-    }
-
     #[DataProvider('formatHighlightData')]
     public function testFormatHighlight(string $sql, string $html): void
     {
-        $this->assertSame($html, $this->formatter->format($sql));
+        $highlighter = new HtmlHighlighter();
+        $formatter   = new SqlFormatter($highlighter);
+
+        $this->assertSame($html, $formatter->format($sql));
     }
 
     #[DataProvider('formatData')]
     public function testFormat(string $sql, string $html): void
     {
-        $formatter = new SqlFormatter(new NullHighlighter());
+        $highlighter = new NullHighlighter();
+        $formatter   = new SqlFormatter($highlighter);
+
         $this->assertSame($html, $formatter->format($sql));
     }
 
     #[DataProvider('highlightData')]
     public function testHighlight(string $sql, string $html): void
     {
-        $this->assertSame($html, $this->formatter->highlight($sql));
+        $highlighter = new HtmlHighlighter();
+        $formatter   = new SqlFormatter($highlighter);
+
+        $this->assertSame($html, $formatter->highlight($sql));
     }
 
     public function testHighlightBinary(): void
     {
-        $sql = 'SELECT "' . pack('H*', 'ed180e98a47a45b3bdd304b798bc5797') . '" AS BINARY';
+        $highlighter = new HtmlHighlighter();
+        $formatter   = new SqlFormatter($highlighter);
 
-        if (defined('ENT_IGNORE')) {
```

Comment:
@derrabus can you please reopen so I can amend this PR and propose this change only - I belive the l61 condition is definitely not needed

Reply:
You're right. Please submit this change as a new PR.

Reply:
Can we please recykle this PR? I will amend/squash.